build.gradle

```diff
@@ -89,7 +89,7 @@ subprojects {
 
                 // TODO: Unreleased dependencies.
                 // These must already be installed in the local maven repository.
-                netty: 'io.netty:netty-codec-http2:4.1.0.Beta5-SNAPSHOT',
+                netty: 'io.netty:netty-codec-http2:4.1.0.Beta6-SNAPSHOT',
```

Comment:
In discussion with @louiscryan on Friday, it seemed we both agreed that we probably wouldn't want to bumping our version of Netty to a SNAPSHOT for a non-essential change. The localhost fix is nice, but probably not enough reason to swap to a SNAPSHOT.